lib/elixir/pages/Guards.md

```diff
@@ -18,6 +18,26 @@ You can find the built-in list of guards [in the `Kernel` module](Kernel.html#gu
 
 The module `Bitwise` also includes a handful of [Erlang bitwise operations as guards](Bitwise.html#guards).
 
+Guards have no concept of "truthy" or "falsey", but instead rely only on strict boolean comparison. If all guards in a clause do not evaluate to the literal boolean value `true`, then the guard will not pass. For example, the following will always return false, even if called with a tuple:
+
+```elixir
+def is_a_tuple?(term) when tuple_size(term), do: true
+def is_a_tuple?(_), do: false
+
+is_a_tuple?({:ok, 1})
+#=> false
+```
+
+To make that guard function correctly you must ensure that the guard evaluates to `true`, like so:
+
+```elixir
+def is_a_tuple?(term) when tuple_size(term) >= 0, do: true
```

Comment:
I am worried this is not a good example because we literally want `is_tuple(...)`. I think a better example would be doing `elem(x, 1)` where the first element is nil or a value, and we need to check for `not is_nil(...)` as just checking `elem(x, 1)` is not enough. Thoughts?

Reply:
I am pushing a fix. :)